### DISCUSSION TITLE
Adding new features to the pipeline

### DISCUSSION THREAD

**Discussion by smcardle** [requarks/wiki#3477] [Help / Questions]
Hi

I am very keen on adding a BPMN 2.0 process viewer into the pipeline.

There is an open source BPMN 2.0 diagrammer called bmpn.io that would suite this very well.
Basically, I would like to upload the BPMN 2.0 XML files as assets and reference then in the page definition as markup. 

bpmn.io is can also create SVG images of the processes from these XML files so there are probably 2 possible ways to integrate in the pipeline.

1. The XML directly visualised
2. Converted to SVG first and then visualised (XML to SVG converter)

Any hints / pointers to documentation regarding extending the pipeline would be welcome

**Comment by rmeekers** [comment]
any updates on this? I'm interested as well
